user_docs/en/userGuide.t2t

```diff
@@ -390,8 +391,10 @@ When moving the review cursor, the System caret does not follow along, so you ca
 However, by default, when the System caret moves, the review cursor follows along.
 This can be toggled on and off.
 
-Note that braille follows the [focus #SystemFocus] and [caret #SystemCaret] by default, rather than object navigation and text review.
-If you want it to follow object navigation and text review instead, you need to [configure braille to be tethered to #BrailleTether] review.
+Note that by default, braille tries to follow automatically both the system focus and system caret as well as the NVDA navigator and the review cursor.
+If you want it to follow the focus and caret only, you need to configure braille to be tethered to focus.
+In this case, braille will not follow the NVDA navigator during object navigation or the review cursor during text review.
+If you want braille to follow only object navigation and text review instead, you need to configure braille to be tethered to review.

```

Comment:
This information belongs in the `#BrailleTether` section, as per line 366 you can replace it with something like `Note [braille following system focus and caret is configurable. #BrailleTether]`

Then make sure that the `#BrailleTether` section contains this information.